ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/MigrationsTest.java

```diff
@@ -108,42 +134,74 @@ public void testApply() throws IOException {
         "CREATE STREAM BAR (A STRING) WITH (KAFKA_TOPIC='BAR', PARTITIONS=1, VALUE_FORMAT='DELIMITED');"
     );
 
-    // This is needed to make sure that the table is fully done being created.
-    // It's a similar situation to https://github.com/confluentinc/ksql/issues/6249
-    assertThatEventually(
-        () -> makeKsqlQuery("SELECT * FROM migration_schema_versions WHERE VERSION_KEY='CURRENT';").size(),
-        is(1)
-    );
-    final int status = MIGRATIONS_CLI.parse("--config-file", configFilePath, "apply", "-a").run();
-    assertThat(status, is(0));
+    // When:
+    final int applyStatus = MIGRATIONS_CLI.parse("--config-file", configFilePath, "apply", "-a").run();
+
+    // Then:
+    assertThat(applyStatus, is(0));
+
+    verifyMigrationsApplied();
+  }
+
+  private void shouldDisplayInfo() {
+    // Given:
+    Logger.getRootLogger().addAppender(logAppender);
+
+    try {
+      // When:
+      final int infoStatus = MIGRATIONS_CLI.parse("--config-file", configFilePath, "info").run();
+
+      // Then:
+      assertThat(infoStatus, is(0));
+
+      verify(logAppender, atLeastOnce()).doAppend(logCaptor.capture());
+      final List<String> logMessages = logCaptor.getAllValues().stream()
+          .map(LoggingEvent::getRenderedMessage)
+          .collect(Collectors.toList());
+      assertThat(logMessages, hasItem(containsString("Current migration version: 2")));
+      assertThat(logMessages, hasItem(matchesRegex(
```

Comment:
This test uses a regex matcher since (1) the timestamp depends on when the test is run, and (2) the timezone used is local.